packages/backend/src/tokens/request.ts

```diff
@@ -96,6 +99,16 @@ export async function authenticateRequest(
       url.searchParams.append(constants.QueryParameters.DevBrowser, authenticateContext.devBrowserToken);
     }
 
+    if (options.organizationSync) {
+      const toActivate = getActivationEntity(requestURL, options.organizationSync);
+      if (toActivate) {
+        const params = getActivationParam(toActivate);
+        Object.entries(params).forEach(([key, value]) => {
+          url.searchParams.append(key, value);
+        });
+      }
+    }
```

Comment:
This allows us to resolve a potential mismatch during the original auth-renewing handshake.